src/main/java/blackjack/view/InputView.java

```diff
@@ -8,39 +9,47 @@
 public class InputView {
 
     private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String NEWLINE = System.getProperty("line.separator");
```

Comment:
```suggestion
    private static final String NEWLINE = System.lineSeparator();
```

별 차이는 없지만.. 해당 기능을 상수 없이 사용할 수 있도록 메서드로 지원하고 있습니다!